app/src/sharedTest/java/org/oppia/android/app/devoptions/DeveloperOptionsActivityTest.kt

```diff
@@ -92,6 +96,7 @@ import org.robolectric.annotation.LooperMode
 import javax.inject.Inject
 import javax.inject.Singleton
 
+// TODO(#3418): Separate DeveloperOptionsActivityTest into activity and fragment test files.
```

Comment:
This sounds like a part of the project so could you assign [this](https://github.com/oppia/oppia-android/issues/3418) issue to yourself or can we do this in this PR only?

Reply:
Also, why are we creating a separate issue for this? Is there any on going work in other PR which are stacked or something?

Reply:
Actually, I have already added the tests to the activity test file. We just have to separate them. But since the tests span across multiple PRs it might cause issues. Also, I still have one pending PR for Milestone1 so I thought we could do this later as I think this does not block the PR.